pom.xml

```diff
@@ -526,4 +526,33 @@
 
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>nodeps</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>2.19.1</version>
+            <configuration>
+              <runOrder>random</runOrder>
+              <reuseForks>false</reuseForks>
+              <excludes>
+                <exclude>Azure*Test</exclude>
+                <exclude>Amazon*Test</exclude>
+                <exclude>Kakadu*Test</exclude>
+                <exclude>Redis*Test</exclude>
+                <exclude>ControlPanelTest</exclude>
```

Comment:
I think the complete list would be:

    <exclude>AmazonS3*Test</exclude>
    <exclude>AzureStorage*Test</exclude>
    <exclude>FfmpegProcessorTest</exclude>
    <exclude>GraphicsMagickProcessorTest</exclude>
    <exclude>ImageMagickProcessorTest</exclude>
    <exclude>KakaduProcessorTest</exclude>
    <exclude>OpenJpegProcessorTest</exclude>
    <exclude>RedisCacheTest</exclude>

I excluded ControlPanelTest as I switched it to use HtmlUnit instead of Firefox+geckodriver, so it no longer has external dependencies (although it's still not working and marked as `@Ignore`).